new/LIT7086QeneBoSadqan.xml

```diff
@@ -0,0 +1,90 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="LIT7086QeneBoSadqan" xml:lang="en" type="work">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title xml:lang="gez" xml:id="t1">ቦ፡ ጻድቃን፡ ውሉደ፡ 
+                    ቤትከ፡ እለ፡ መነኑ፡ ዓለመ፡ ወዕለ፡ ኤሉ፡ አድባረ፤</title>
```

Comment:
Perhaps it is better to finish t1 with ፡ since this is the title of the poem. The proper punctuation will be provided in "Text".

Reply:
I have been using the "line-final" punctuation marks regularly in "my" qene records. I could change them all, if this is better. Do you agree, @eu-genia ?